ios/Exponent/Kernel/DevSupport/EXKernelDevKeyCommands.m

```diff
@@ -122,6 +166,25 @@ + (void)initialize
   RCTSwapInstanceMethods([UIResponder class],
                          @selector(keyCommands),
                          @selector(EX_keyCommands));
+
+  SEL originalKeyboardSelector = NSSelectorFromString(@"handleKeyUIEvent:");
+  RCTSwapInstanceMethods([UIApplication class],
+                         originalKeyboardSelector,
+                         @selector(EX_handleKeyUIEventSwizzle:));
+}
+
++(void)handleKeyboardEvent:(UIEvent *)event
+{
+  static NSTimeInterval lastCommand = 0;
+
+  if (event._isKeyDown) {
+    if (CACurrentMediaTime() - lastCommand > 0.5) {
```

Comment:
lastCommand is always 0

Reply:
It isn't, only on the first call 😉 static variables in ObjC are persistent in subsequent method calls

Reply:
it was fixed since i commented :O when i commented we never set `lastCommand` again

Reply:
actually nevermind i don't think it was 🤔 

https://github.com/expo/expo/blob/06214f455fd54c0014fca8f0c8cc544c3f758ab9/ios/Exponent/Kernel/DevSupport/EXKernelDevKeyCommands.m#L172-L184

what am i missing here @tsapeta?